bz-mill/config/definitions/map_namespace_to_query

```diff
@@ -0,0 +1,2 @@
+http://red.ht/2oiunCH bz.gmeno.ceph.new.customer.24hrs
+http://red.ht/2paAywM bz.gmeno.ceph.any.needinfo.24hrs
```

Comment:
is `24hrs` a query or just part of the namespace? if the latter, you might want to skip it because you get that for free with gradana